zulipterminal/ui_tools/boxes.py

```diff
@@ -172,41 +172,11 @@ def send_stop_typing_status(self) -> None:
             self.idle_status_tracking = False
             self.sent_start_typing_status = False
 
-    def private_box_view(
-        self,
-        *,
-        recipient_user_ids: Optional[List[int]] = None,
-    ) -> None:
+    def _setup_common_private_compose(self) -> None:
         self.set_editor_mode()
 
         self.compose_box_status = "open_with_private"
 
-        if recipient_user_ids:
-            self._set_regular_and_typing_recipient_user_ids(recipient_user_ids)
-            self.recipient_emails = [
-                self.model.user_id_email_dict[user_id]
-                for user_id in self.recipient_user_ids
-            ]
-            recipient_info = ", ".join(
-                [
-                    f"{self.model.user_dict[email]['full_name']} <{email}>"
-                    for email in self.recipient_emails
-                ]
-            )
-        else:
-            self._set_regular_and_typing_recipient_user_ids(None)
-            self.recipient_emails = []
-            recipient_info = ""
-
-        self.send_next_typing_update = datetime.now()
-        self.to_write_box = ReadlineEdit("To: ", edit_text=recipient_info)
-        self.to_write_box.enable_autocomplete(
-            func=self._to_box_autocomplete,
-            key=primary_key_for_command("AUTOCOMPLETE"),
-            key_reverse=primary_key_for_command("AUTOCOMPLETE_REVERSE"),
-        )
-        self.to_write_box.set_completer_delims("")
```

Comment:
If we shouldn't be sending typing updates for editing, I suspect that could be clearer in the API documentation. If you're sure this is the behavior we want, posting to that stream that you think that's the case would be useful, or even a quick PR for it.

Since we're splitting the private-compose and private-edit UI it certainly makes it simpler to separate this behavior in this way.